abracadabra/src/writer/persistency_manager.hh

```diff
@@ -14,13 +14,13 @@ public:
     persistency_manager();
     virtual ~persistency_manager() override { this->close_file(); };
 
-    virtual G4bool Store(const G4Event*)           override;
-    virtual G4bool Store(const G4Run*)             override ;
-    virtual G4bool Store(const G4VPhysicalVolume*) override;
+    G4bool Store(const G4Event*)           override;
+    G4bool Store(const G4Run*)             override;
+    G4bool Store(const G4VPhysicalVolume*) override { return true; }
```

Comment:
Is the meaning of these return values documented anywhere?

I'm guessing that `false` means failed to read/write, or maybe that nothing is supposed to be read/written, while `true` means success ... but I haven't found any docs that explain it.